nmdc_automation/re_iding/scripts/re_id_tool.py

```diff
@@ -157,10 +175,12 @@ def update_study(ctx, legacy_study_id, nmdc_study_id,  mongo_uri, identifiers_fi
     updates = {
         "study_set": {},
         "biosample_set": {},
-        "omics_processing_set": {}
+        "omics_processing_set": {},
+        "data_object_set": {},
+        "metabolomics_analysis_activity_set": {},
```

Comment:
Should this list include nom_analysis_activity_set?

Reply:
Yes -

Reply:
https://github.com/microbiomedata/nmdc_automation/pull/116/commits/54b01bc0144d7d1486bd471a49d007ff8d063af3